system/libraries/Session/drivers/Session_database_driver.php

```diff
@@ -89,7 +89,7 @@ public function __construct(&$params)
 		{
 			throw new Exception('Query Builder not enabled for the configured database. Aborting.');
 		}
-		elseif ($this->_db->pconnect)
+		elseif (! $this->_db->pconnect)
```

Comment:
Yes. Think this should be for codeigniter classes I found that My problem in activating database session is this part current line set an exception for each time that db connection is ok.